acceptance/tests/language/invoke_pp_function_from_dependent_module.rb

```diff
@@ -0,0 +1,81 @@
+test_name 'Puppet-language functions can be called from inside an another module' do
+
+  mini_func = 'function jenny::mini($a, $b) {if $a <= $b {$a} else {$b}}'
+  maxi_func = 'function jenny::nested::maxi($a, $b) {if $a >= $b {$a} else {$b}}'
```

Comment:
just an idea, but if we have two functions, maybe the second should do some string operations?